### ISSUE TITLE
speccy lint --skip not skipping 0.8.1

### ISSUE THREAD

**Issue by jblazek** [wework/speccy#151]
When using the current version of speccy 0.8.1, the --skip (-s) option for linting doesn't seem to do anything.

Previously, this would ignore lint rules by rule id.

**Working**
```bash
npm install -g speccy@0.7.3
speccy lint https://raw.githubusercontent.com/OAI/OpenAPI-Specification/master/examples/v3.0/petstore.yaml --skip openapi-tags --skip info-contact
Specification is valid, with 0 lint errors
```

**Not Working**
```bash
npm install -g speccy@0.8.1
speccy lint https://raw.githubusercontent.com/OAI/OpenAPI-Specification/master/examples/v3.0/petstore.yaml --skip openapi-tags --skip info-contact
Finished resolution!
Specification contains lint errors: 2
#/info  R: info-contact  D: info object should contain contact object
<...snip...>
#/  R: openapi-tags  D: openapi object should have non-empty tags array
<...snip>
```

**Comment by bburtin** [comment]
I hit this too.  I don't think it's just `-s`.  I couldn't get `-r` to work either.  I think that `cleanObject()` in *config.js* is busted.  If I remove the call to `cleanObject()`, speccy seems to do what it's supposed to.
```
        nconf.add('supplied', {
            type: 'literal',
            // store: this.cleanObject(supplied),
            store: supplied
        });
```

**Comment by philsturgeon** [comment]
Fixed in #159. 

Run `npm install speccy@0.8.2-1` to get this immediately, but I'll be tagging v0.8.2 proper soon.
